ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -122,6 +124,7 @@ def enable(cls, flush_interval=None, tracer=None, dogstatsd_url=None):
         with cls._lock:
             if cls._instance is not None:
                 return
+            flush_interval = flush_interval or _get_default_interval()
```

Comment:
nit, but if `flush_interval` is 0 you'll ignore it… which is fine but not really nice. 😁

Reply:
So `0` would mean "use env var or default" in this case. Is this what we want?

Reply:
Changed it so we handle `flush_interval=0` as effectively disabling the flush call.